docs/release_flow/linux.md

```diff
@@ -0,0 +1,16 @@
+
+
+# Install Linux Preview-Releases: 
```

Comment:
What is a "Preview-Release"?

Reply:
Is it kinda like a nightly?

Reply:
Sorry poor wording. I meant the release candidate :D

Reply:
Oh OK! Mm, then I think it makes sense to push it to the staging repository. Is this for testing before uploading releases to the product delivery repository?

Reply:
Aaah yea i just realized that this is not going to do what i expected it to :D I wrote this way later than when I actually wrote the transform. Given this will only publish to the `prod` repo, 
i probably should restrict it to `ship-client`, right?

https://github.com/mozilla-mobile/mozilla-vpn-client/pull/8593/files#diff-6b9dd2081b39f9e6fe18c9578d3cf95b68e9b7ef232312fbcc0e5c24dc412b6eR44-R47

I think it's fine if this one only goes to prod, somehow i wrongly remembered, `promote-client` on the vpn-repo also would push to the stage repo, therefore contain the RC's for QA 😅